app/views/layouts/application.html.slim

```diff
@@ -0,0 +1,12 @@
+doctype html
```

Comment:
Can we please stick to `html.erb` for templates?

I would prefer not to have identation-based markup languages anywhere; they are more pleasant to write, but are really terrible when you have to maintain them: git merges gone wrong might result in far-from-obvious failure modes.